<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblUsu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNome">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nível de Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do Usuario:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFuncionarios">
    <property name="title">
     <string>Funcionarios</string>
    </property>
    <addaction name="actionCadastroFun"/>
    <addaction name="actionEdFun"/>
    <addaction name="actionDELFunc"/>
    <addaction name="actionListarFunc"/>
   </widget>
   <widget class="QMenu" name="menuPedidos">
    <property name="title">
     <string>Pedidos</string>
    </property>
    <addaction name="actionCadastroPed"/>
    <addaction name="actionEdPedido"/>
    <addaction name="actionDELPedido"/>
    <addaction name="actionListarPed"/>
   </widget>
   <widget class="QMenu" name="menuAdministrador">
    <property name="title">
     <string>Administrador</string>
    </property>
    <addaction name="actionCadastroAdm"/>
    <addaction name="actionEdAdmin"/>
    <addaction name="actionDELAdmin"/>
    <addaction name="actionListarAdmin"/>
   </widget>
   <widget class="QMenu" name="menuCliente">
    <property name="title">
     <string>Cliente</string>
    </property>
    <addaction name="actionCadastroCli"/>
    <addaction name="actionEdCli"/>
    <addaction name="actionDELCli"/>
    <addaction name="actionListarCli"/>
   </widget>
   <widget class="QMenu" name="menuProduto">
    <property name="title">
     <string>Produto</string>
    </property>
    <addaction name="actionCadastroPro"/>
    <addaction name="actionEdProd"/>
    <addaction name="actionDELProd"/>
    <addaction name="actionListarProd"/>
   </widget>
   <widget class="QMenu" name="menuGanhos">
    <property name="title">
     <string>Ganhos</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionMes"/>
    <addaction name="actionAnual"/>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Login</string>
    </property>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuCliente"/>
   <addaction name="menuFuncionarios"/>
   <addaction name="menuPedidos"/>
   <addaction name="menuAdministrador"/>
   <addaction name="menuProduto"/>
   <addaction name="menuGanhos"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCadastroCli">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionEdCli">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionDELCli">
   <property name="text">
    <string>Exclusão</string>
   </property>
  </action>
  <action name="actionCadastroFun">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionEdFun">
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionCadastroPed">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionEdi_o_3">
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionCadastroAdm">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionCadastroPro">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionEdPedido">
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionDELFunc">
   <property name="text">
    <string>Exclusão</string>
   </property>
  </action>
  <action name="actionDELPedido">
   <property name="text">
    <string>Exclusão</string>
   </property>
  </action>
  <action name="actionEdAdmin">
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionDELAdmin">
   <property name="text">
    <string>Exclusão</string>
   </property>
  </action>
  <action name="actionEdProd">
   <property name="text">
    <string>Edição</string>
   </property>
  </action>
  <action name="actionDELProd">
   <property name="text">
    <string>Exclusão</string>
   </property>
  </action>
  <action name="actionMes">
   <property name="text">
    <string>Mensal</string>
   </property>
  </action>
  <action name="actionAnual">
   <property name="text">
    <string>Anual</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionListarCli">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionListarFunc">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionListarPed">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionListarAdmin">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionListarProd">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionCadastroCli</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCli()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCadastroFun</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCadastroAdm</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCadastroPro</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showProd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCadastroPed</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPedido()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdCli</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCliED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDELCli</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showCliDEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdFun</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFuncED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDELFunc</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFuncDEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdPedido</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPedidoED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDELPedido</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPedidoDEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdAdmin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAdminED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDELAdmin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAdminDEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdProd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showProdED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDELProd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showProdDEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMes</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showGanhosMes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAnual</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showGanhosAno()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSair</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListarCli</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListarCli()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListarFunc</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListarFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListarPed</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListarPed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListarProd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListarProd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionListarAdmin</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListarAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showCli()</slot>
  <slot>showFunc()</slot>
  <slot>showAdmin()</slot>
  <slot>showProd()</slot>
  <slot>showPedido()</slot>
  <slot>showPedidoED()</slot>
  <slot>showCliED()</slot>
  <slot>showFuncED()</slot>
  <slot>showAdminED()</slot>
  <slot>showCliDEL()</slot>
  <slot>showPedidoDEL()</slot>
  <slot>showFuncDEL()</slot>
  <slot>showAdminDEL()</slot>
  <slot>showProdED()</slot>
  <slot>showProdDEL()</slot>
  <slot>showGanhosMes()</slot>
  <slot>showGanhosAno()</slot>
  <slot>showLogin()</slot>
  <slot>showListarCli()</slot>
  <slot>showListarPed()</slot>
  <slot>showListarProd()</slot>
  <slot>showListarAdmin()</slot>
  <slot>showListarFunc()</slot>
 </slots>
</ui>
